lib/Subscriber/Core/RegistratorSubscriber.php

```diff
@@ -83,7 +84,7 @@ public function configureOptions(ConfigureOptionsEvent $event)
      *
      * @param HydrateEvent $event
      */
-    public function handleDefaultLocale(HydrateEvent $event)
+    public function handleDefaultLocale(AbstractMappingEvent $event)
```

Comment:
Not having a locale at all means every thing needing to consider the
abscence of the locale. If you take for granted the presence of the
locale, then you have no problem.

What is the issue? If you use a default locale you do not need to make
checks like "that". Everybody is happy.

On Tue, Apr 19, 2016 at 07:43:42AM -0700, Daniel Rotter wrote:
>    In [1]lib/Subscriber/Core/RegistratorSubscriber.php:
> 
>  > @@ -83,7 +84,7 @@ public function configureOptions(ConfigureOptionsEvent $event)
>  >       *
>  >       * @param HydrateEvent $event
>  >       */
>  > -    public function handleDefaultLocale(HydrateEvent $event)
>  > +    public function handleDefaultLocale(AbstractMappingEvent $event)
> 
>    The fact that we use [2]checks like this makes this change of the document
>    manager incompatible with the current state of Sulu. Also, it still
>    doesn't make any sense to pass a locale when we e.g. move a page, since
>    this change is valid for all localizations, and not only a single one.
>    This is just confusing and screams for unexpected behavior.
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [3]view it on GitHub
> 
>    Reverse link: [4]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu-document-manager/pull/72#discussion_r60243953
>    2. https://github.com/sulu/sulu/blob/7f745ac7c5617bfcf3a92de53871e03829c08a19/src/Sulu/Component/Content/Document/Subscriber/StructureSubscriber.php#L202
>    3. https://github.com/sulu/sulu-document-manager/pull/72/files/8e599707818caabeeea23d162b9a19cd78a0b234#r60243953
>    4. https://github.com/sulu/sulu-document-manager/pull/72/files/8e599707818caabeeea23d162b9a19cd78a0b234#r60243953